requirements.txt

```diff
@@ -0,0 +1,15 @@
+-c constraints.txt
+
+Sphinx
+jsx-lexer
+lesscpy
+linkify-it-py
+myst-parser
+sphinx-autobuild
+sphinx-book-theme
+sphinx-copybutton
+sphinx-design
+sphinx-sitemap
+sphinx-togglebutton
+sphinxcontrib-spelling
```

Comment:
We switched to Vale instead of this extension. Documentation has the latest configuration. See also https://6.docs.plone.org/contributing/documentation/setup-build.html#vale

```suggestion
```

Reply:
same